docs/source/models/concept_models/vivarium_nutrition_optimization/concept_model.rst

```diff
@@ -57,14 +57,77 @@ Documents that contain information specific to the subcomponents of this simulat
 2.0 Modeling aims and objectives
 ++++++++++++++++++++++++++++++++
 
+**Objective:** For a given fiscal envelope, determine the optimal funding allocation across select nutrition-related intervention products for maternal and child health. "Optimal" is defined with respect to some specified outcome such as the funding allocation that minimizes DALYs or maximizes the number of alive, non-stunted children.
 
+.. note::
+  
+  We must be careful in optimizing for "minimizing" the prevalence of some risk such as child growth failure. This is because prevalence of such risks could be decreased via decreasing incidence (positive health impact) *or* via increasing risk-related mortality (negative health impact). That is why we choose to *maximize* the number of alive non-stunted children rather than minimize the prevalence of child growth failure.
```

Comment:
It seems like minimizing **DALYs** due to child growth failure would be more consistent with how we usually model things? Maximizing the number of alive non-stunted children will incentivize our optimization to increase fertility, and will place no importance on improving the mortality rate among those with stunting.

Reply:
Interesting!

I don't know the details, but given that we have some pregnancy-related interventions, it still seems like we could end up "improving stunting" by increasing the number of live births in the sim. For example, if the budget was large enough that we had already maxed out all the direct stunting treatment. I wonder if the Optima paper had this issue as well?

Once we get there, it might be interesting to look at both (minimizing DALYs due to stunting, maximizing non-stunted children), which would directly show whether this is an issue. Both measures of impact will be in our outputs, I assume, so it should be pretty straightforward to check it.

Reply:
Ah, I see what you're saying. Yeah I agree that would be an interesting comparison. FWIW, I do expect the fertility impacts to be tiny (stillbirths are like <1% of live births in Ethiopia and we reduce the risk of stillbirth by like 10% or something). I also think we would need to be clever about tracking CGF-attributable DALYs in the sim... typically this rate is estimated at a population rather than individual level, so we'd have to do some extra math to decide whether someone who is stunted and dies of a stunting-related cause would have still died of that cause if they were not stunted at the time of death. We can definitely do so, but I do think it would be some extra observer logic.

This makes me think though that we need to more carefully think through stillbirth dynamics more generally. By minimizing DALYs we may deprioritize interventions that reduce stillbirths (if we keep with GBD practice of not assigning stillbirths any DALYs). In this case, it may actually be "better" to maximize the number of alive non-stunted children (averting a stillbirth is good! But isn't actually directly related to stunting...). Without stillbirths, I think the approaches will be equivalent, so I think starting with how we want to consider stillbirths will be important in guiding our next steps here...

I am leaning towards assigning stillbirths YLLs. @aflaxman, do you have any concerns with that approach?

Reply:
Note: "increasing the number of live births" does not (in principle) depend only on stillbirth dynamics. An intervention could also increase the number of live births by averting miscarriages, averting pregnancy-related deaths which prevent future pregnancies, or even directly increasing the fertility rate. I don't know if any of our interventions do these things, however.

Reply:
Yes, true! None of our modeled interventions avert miscarriages or directly affect the fertility rate. Also, we have designed our sim to only evaluate single pregnancies/births/childhoods in a closed cohort fashion. In other words our simulated timeframe does not allow for repeat pregnancies and therefore we won't increase fertility due to averting pregnancy-related deaths within our sim, although this could happen in real life - so a definite limitation here although this would be expected to have a small effect because death during childbirth is still a rare outcome.

I'll be sure to document these things that we're able to make simplifying assumptions around for now that we would want to revisit if we ever added interventions that would break these assumptions.